suite/sanity/loss_recover.go

```diff
@@ -104,9 +104,11 @@ func lossAndRecovery(p interface{}) (gravity.TestFunc, error) {
 			g.Logger().WithFields(logrus.Fields{"roles": roles, "nodes": nodes}).
 				Info("roles after expand")
 
-			g.OK("remove old node", g.RemoveNode(nodes, removed))
+			g.OK("remove old node", g.RemoveNodes(nodes, []gravity.Gravity{removed}))
```

Comment:
Nit: Not really sure why it says "old" node, I'd just say "a" node.

Reply:
Cleaned up in 1ed898d.